tests/metrics/test_metastatus_lib.py

```diff
@@ -689,6 +689,48 @@ def test_get_resource_utilization_per_slave():
     )
 
 
+def test_calculate_resource_utilization_for_slaves():
+    fake_slaves = [
+        {
+            'id': 'somenametest-slave2',
+            'hostname': 'test2.somewhere.www',
+            'resources': {
+                'cpus': 500,
+                'disk': 200,
+                'mem': 750,
+            },
+            'reserved_resources': {},
+            'attributes': {
+                'habitat': 'somenametest-habitat-2',
+            },
+        },
+    ]
+    tasks = [
+        {
+            'resources': {
+                'cpus': 10,
+                'mem': 10,
+                'disk': 10
+            },
+            'state': 'TASK_RUNNING'
+        },
+        {
+            'resources': {
+                'cpus': 10,
+                'mem': 10,
+                'disk': 10
+            },
+            'state': 'TASK_RUNNING'
```

Comment:
One of these should be a terminal state right? I guess we'll see if I'm right in a mo when travis finishes...